main.tf

```diff
@@ -1,236 +1,428 @@
+  }
+
+  depends_on = [
+    aws_apigatewayv2_api.this,
+    aws_apigatewayv2_route.this,
+    aws_apigatewayv2_integration.this,
+  ]
+
   lifecycle {
     create_before_destroy = true
   }
 }
 
-# Authorizers
-resource "aws_apigatewayv2_authorizer" "this" {
-  for_each = { for k, v in var.authorizers : k => v if var.create && var.create_routes_and_integrations }
-
-  api_id = aws_apigatewayv2_api.this[0].id
+################################################################################
+# Stage Access Logs - Log Group
+################################################################################
 
-  authorizer_type                   = try(each.value.authorizer_type, null)
-  identity_sources                  = try(flatten([each.value.identity_sources]), null)
-  name                              = try(each.value.name, null)
-  authorizer_uri                    = try(each.value.authorizer_uri, null)
-  authorizer_payload_format_version = try(each.value.authorizer_payload_format_version, null)
-  authorizer_result_ttl_in_seconds  = try(each.value.authorizer_result_ttl_in_seconds, null)
-  authorizer_credentials_arn        = try(each.value.authorizer_credentials_arn, null)
-  enable_simple_responses           = try(each.value.enable_simple_responses, null)
+resource "aws_cloudwatch_log_group" "this" {
+  for_each = { for k, v in { "this" = var.stage_access_log_settings } : k => v if local.create_stage && try(v.create_log_group, true) }
 
-  dynamic "jwt_configuration" {
-    for_each = length(try(each.value.audience, [each.value.issuer], [])) > 0 ? [true] : []
+  name              = coalesce(each.value.log_group_name, "/aws/apigateway/${var.name}/${replace(var.stage_name, "$", "")}")
+  retention_in_days = each.value.log_group_retention_in_days
+  kms_key_id        = each.value.log_group_kms_key_id
+  skip_destroy      = each.value.log_group_skip_destroy
+  log_group_class   = each.value.log_group_class
 
-    content {
-      audience = try(each.value.audience, null)
-      issuer   = try(each.value.issuer, null)
-    }
-  }
+  tags = merge(var.tags, each.value.log_group_tags)
 }
 
-# VPC Link (Private API)
+################################################################################
+# VPC Link
+################################################################################
+
 resource "aws_apigatewayv2_vpc_link" "this" {
-  for_each = { for k, v in var.vpc_links : k => v if var.create && var.create_vpc_link }
+  for_each = { for k, v in var.vpc_links : k => v if var.create }
 
-  name               = try(each.value.name, each.key)
-  security_group_ids = each.value["security_group_ids"]
-  subnet_ids         = each.value["subnet_ids"]
+  name               = coalesce(each.value.name, each.key)
+  security_group_ids = each.value.security_group_ids
+  subnet_ids         = each.value.subnet_ids
 
-  tags = merge(var.tags, var.vpc_link_tags, try(each.value.tags, {}))
+  tags = merge(var.tags, var.vpc_link_tags, each.value.tags)
```

Comment:
```suggestion
  tags = merge(var.tags, var.vpc_link_tags, try(each.value.tags, {}))
```

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-apigateway-v2/pull/54/commits/d6e694c0e73bf8c7fc7d145e074c3d562dc36848